google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/sdk/CloudSdkService.java

```diff
@@ -40,7 +44,46 @@ public static CloudSdkService getInstance() {
 
   public abstract void setSdkHomePath(String path);
 
-  public abstract Set<CloudSdkValidationResult> validateCloudSdk(@NotNull Path pathToCloudSdk);
+  protected abstract Set<CloudSdkValidationResult> validateCloudSdk(Path path);
+
+  /**
+   * Checks if the stored path contains a valid Cloud SDK.
+   */
+  public Set<CloudSdkValidationResult> validateCloudSdk() {
```

Comment:
They don't depend on any `DefaultCloudSdkService` members or methods, unlike `validateCloudSdk(Path)` which depends on `buildCloudSdkWithPath()` and `isCloudSdkVersionSupported()`.